docsite/docs/Reference/variables-and-functions.md

```diff
@@ -70,26 +93,74 @@ from sqlalchemy.types import Integer
 write_to_source(df, 'source', 'table', dtype={'value': Integer()})
 ```
 
-### `write_to_source` modes of _writing_
+### `write_to_model` function
+
+You have to define the target source in your schema and use it in fal.
+This operation overwrites the existing relation by default and should only be used targetting tables, not views.
+
+When used in `fal flow run` or `fal run`, it does not accept a model name, since it only operates on the associated model
+
+```python
+df = faldbt.ref('stg_zendesk_ticket_data')
+df = add_zendesk_metrics_info(df)
+
+# Upload a `pandas.DataFrame` back to the data warehouse
+write_to_model(df)
+```
```

Comment:
"You have to define the target source in your schema and use it in fal." 

is this right? Maybe i am misunderstanding what this means. 
I think here just explain the happy case, follow it by the example and then go into more detail.

---
This operation overwrites an existing relation. For example, if the script below is attached to the `stg_zendesk_ticket_data` model, `write_to_model` operation updates the relation represented by the `stg_zendesk_ticket_data` dbt model.

```python
df = faldbt.ref('stg_zendesk_ticket_data')
df = add_zendesk_metrics_info(df)
# Upload a `pandas.DataFrame` back to the data warehouse
write_to_model(df)
```
`write_to_model` should only be used targetting tables, not views.

But when importing ....

Reply:
Check it out again